src/interfaces.ts

```diff
@@ -124,6 +124,23 @@ export interface Plugin {
   reset(): Promise<boolean>;
 }
 
+/**
+ * Arguments to be passed to `node:worker_threads.Worker` when `runInWorker: true`.
+ */
+export interface NodeWorkerArgs {
+  name?: string;
+  execArgv?: string[];
+  argv?: string[];
+  env?: Record<string, string>;
+  resourceLimits?: {
+    maxOldGenerationSizeMb?: number;
+    maxYoungGenerationSizeMb?: number;
+    codeRangeSizeMb?: number;
+    stackSizeMb?: number;
+  };
+  [k: string]: any;
```

Comment:
This is pretty much copied from Node's API, but neither `Parameters<Worker>` nor `ConstructorParameters<Worker>` worked so I've manually listed them here.